rpcs3/Emu/Audio/XAudio2/XAudio28Thread.cpp

```diff
@@ -57,6 +65,8 @@ void XAudio2Thread::xa28_destroy()
 		s_tls_xaudio2_instance->StopEngine();
 		s_tls_xaudio2_instance->Release();
 	}
+
+	CoUninitialize();
```

Comment:
Strange, I though Xaudio 2.8 does `CoXXX` stuff automatically (because it's nearly impossible to correctly uninitilize xaudio 2.7 running in several threads). http://stackoverflow.com/questions/16049677/xaudio2-access-violation-exception-when-unloaded (and, irc doplhin doesn't have any problems without `Co`).
Btw, I don't think that we `FreeLibrary` anywhere, not sure if it's good.